test/distributed/org/apache/cassandra/distributed/impl/InstanceConfig.java

```diff
@@ -113,6 +114,45 @@ private InstanceConfig(int num,
                 .set("index_summary_capacity", "50MiB")
                 .set("counter_cache_size", "50MiB")
                 .set("key_cache_size", "50MiB");
+        if (CassandraRelevantProperties.DTEST_JVM_DTESTS_USE_LATEST.getBoolean())
+        {
+            // TODO: make this load latest_diff.yaml or cassandra_latest.yaml
```

Comment:
I would add a note in build.xml or some testing README about this. It's going to be very confusing changing yamls and seeing no effect as verything is hardcoded here.

Reply:
Done

Reply:
Do we want to make that param a String instead of a Boolean to open the door to more configs?